articles/article-025-we-all-hate-bugs.md

```diff
@@ -0,0 +1,22 @@
+<!-- BEGIN TITLE -->
+# We all hate Bugs
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+Bugs are something nobody wants. They creep into your house or your code without notice. There they eat your resources and clog your sink. 
+They hide in corners and behind large furniture and are most of the times hard to find. But whether in the household or in the software code, if you find a bug that was bothering you for a long time the feeling of success is rewarding.
+A bug in software is usually a behavior of the software which is not intended. Sometimes they are very hard to find and can be attributed to one single character that was wrongly placed. Other times they are structural mistakes that lead to a "bridge collapse". The act of finding bugs is called Debugging. Like pest control, the code is searched and cleaned from every bug that would influence the behavior of the software in a negative way.
```

Comment:
The bridge collapse is difficult to understand. But the article would work without the distinction between the two types of bugs (single character / structural). Also, is a structural bug really a bug or just a design flaw?